Projects/Feature/Sources/ObservedObject/MainObservable.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  MainObservableObject.swift
+//  Feature
+//
+//  Created by 한지석 on 10/23/23.
+//  Copyright © 2023 com.kozi. All rights reserved.
+//
+
+import SwiftUI
+import SwiftData
+import Core
```

Comment:
```suggestion
import Core
import SwiftData
import SwiftUI
```
import 순서 바꾸시면 좋을 듯 합니다!